src/main/java/com/pinterest/secor/common/KafkaClient.java

```diff
@@ -5,9 +5,9 @@
  * The ASF licenses this file to You under the Apache License, Version 2.0
  * (the "License"); you may not use this file except in compliance with
  * the License.  You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
+ * <p>
```

Comment:
Yes, when I run mvn package, these lines are getting added. Should I keep it or remove?